### PR TITLE
Adding initial Vaxrank (detailed) docs for ReadTheDocs integration.

### PR DESCRIPTION
**Pull request by julia326**

### CODE REVIEW

File: docs/Makefile [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+# Put it first so that "make" without argument is like "make help".
+help:
+	@$(SPHINXBUILD) -M help "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
+
+.PHONY: help Makefile
+
+# Catch-all target: route all unknown targets to Sphinx using the new
+# "make mode" option.  $(O) is meant as a shortcut for $(SPHINXOPTS).
+%: Makefile
+	@$(SPHINXBUILD) -M $@ "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
```
Comment (iskandr) [inline docs/Makefile]: Does this get invoked by RtD?

File: docs/conf.py [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+extensions = []
+
+# Add any paths that contain templates here, relative to this directory.
+templates_path = ['_templates']
+
+# The suffix(es) of source filenames.
+# You can specify multiple suffix as a list of string:
+#
+# source_suffix = ['.rst', '.md']
+source_suffix = '.rst'
```
Comment (iskandr) [inline docs/conf.py]: Maybe an obvious question, but why not MD?

File: docs/Makefile (hunk 1 above)
Comment (julia326) [reply docs/Makefile]: presumably yeah; this was auto-generated using the Sphinx instructions at https://docs.readthedocs.io/en/latest/getting_started.html

File: docs/conf.py (hunk 2 above)
Comment (julia326) [reply docs/conf.py]: that same manual, RtD docs express a preference for RST though they support both, so I went with that

**Pull request merged**